src/components/IndividualArticle-components/Article.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import { getArticleById } from "../../api-calls/api-calls";
+import Lottie from "lottie-react";
+import cogLoading from "../../assets/loading.json";
+
+export const Article = ({ articleid }) => {
+  const [getArticle, setArticle] = useState();
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getArticleById(articleid)
+      .then(({ data: { article } }) => {
+        setIsLoading(false);
+        setArticle(article);
+      })
+      .catch((err) => {
+        setIsLoading(false);
+        console.log(err);
+      })
+      .finally(() => {
```

Comment:
Interesting find!! I like this. You are repeating yourself though by setting false on line 13. As the finally section happens regardless of the result you could remove line 13.